.eslintrc.js

```diff
@@ -5,4 +5,10 @@ module.exports = {
     'jsx-a11y',
     'import',
   ],
+  rules: {
+    "function-paren-newline": "off",
+    "no-restricted-syntax": "off",
+    "react/no-deprecated": "off",
+    "import/no-anonymous-default-export": "off",
```

Comment:
@roblevintennis This rule does the same thing as our custom lint rule for `export default { ... }`. I went with this since it is more configurable (etc, etc).

If you remove this line, CI will fail. However, it passes in https://github.com/mavenlink/brainstem-redux/pull/68/ since we fixed the bad exports. :+1: